routemaster/config/visualisation.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<style>
+body {
+  font: 14px helvetica neue, helvetica, arial, sans-serif;
+  margin: 0;
+}
+#cy {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  left: 0;
+  top: 3.5em;
+}
+#controls {
+  background-color: #f0f0f0;
+  margin: 0;
+}
+#controls div {
+  display: inline-block;
+  padding: 0.5em;
+  margin-right: 0.2em;
+}
+#spacing-factor-input-container {
+  top: 0.5em;
+  position: relative;
+}
+.muted {
+  color: #666666;
+}
+.action {
+  background-color: orange;
+}
+.gate {
+  background-color: lightBlue;
+}
+</style>
+<title>Visualisation of state machine {{ state_machine_name }}</title>
+</head>
+<body>
+  <script type="text/javascript">
+    var state_machine_config = {{ state_machine_config|tojson }};
+  </script>
```

Comment:
It would be great to move the scripts (and the above CSS) to separate files.

Reply:
I quite like that it's all a single file at the moment; it makes editing fairly easy.